src/main/java/frc/robot/RobotContainer.java

```diff
@@ -157,8 +177,24 @@ public RobotContainer() {
     
     // this.colorSensorSubsystem.setDefaultCommand(colorSensorCommand);   <--- Causes an error right now
 
+    // Autonomous commands
+    this.m_autoDriveForwardCommand = new DriveStraightCommand(navxGyroSubsystem, drivetrainSubsystem, m_blinkinSubsystem, ()-> 0.3 , ()-> 0.3, ()-> 1);
+    this.m_autoDriveBackwardCommand = new DriveStraightCommand(navxGyroSubsystem, drivetrainSubsystem, m_blinkinSubsystem, ()-> -0.3 , ()-> -0.3, ()-> 1);
+    this.m_autoDriveAndScoreCone = new DriveAndScoreConeCommand(navxGyroSubsystem, drivetrainSubsystem, m_blinkinSubsystem, m_armSubsystem, m_grabberSubsystem);
+    this.m_autoDriveAndScoreCube = new DriveAndScoreCubeCommand(navxGyroSubsystem, drivetrainSubsystem, m_blinkinSubsystem, m_armSubsystem, m_grabberSubsystem);
+
+    this.m_auto_command_chooser.setDefaultOption("Drive backwards for 2 seconds", m_autoDriveBackwardCommand.withTimeout(2));
+    this.m_auto_command_chooser.addOption("Drive forward 2 seconds", m_autoDriveForwardCommand.withTimeout(2));
+    m_auto_command_chooser.addOption("Drive forward and score cube", m_autoDriveAndScoreCube);
```

Comment:
```suggestion
    m_auto_command_chooser.addOption("Score cube then drive backward", m_autoDriveAndScoreCube);
```
Change labels here so they are not confusing to the driver